pkgs/by-name/dm/dmg2img/package.nix

```diff
@@ -1,33 +1,50 @@
-{ lib, stdenv, fetchurl, zlib, bzip2, openssl, fetchpatch }:
+{
+  bzip2,
+  fetchFromGitHub,
+  lib,
+  openssl,
+  stdenv,
+  unstableGitUpdater,
+  zlib,
+}:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "dmg2img";
-  version = "1.6.7";
+  version = "1.6.7-unstable-2020-12-27";
 
-  src = fetchurl {
-    url = "http://vu1tur.eu.org/tools/dmg2img-${version}.tar.gz";
-    sha256 = "066hqhg7k90xcw5aq86pgr4l7apzvnb4559vj5s010avbk8adbh2";
+  src = fetchFromGitHub {
+    owner = "Lekensteyn";
+    repo = "dmg2img";
+    rev = "a3e413489ccdd05431401357bf21690536425012";
+    hash = "sha256-DewU5jz2lRjIRiT0ebjPRArsoye33xlEGfhzd4xnT4A=";
   };
 
-  buildInputs = [ zlib bzip2 openssl ];
-
-  patches = [
-    (fetchpatch {
-      url = "https://raw.githubusercontent.com/Homebrew/formula-patches/85fa66a9/dmg2img/openssl-1.1.diff";
-      sha256 = "076sz69hf3ryylplg025vl8sj991cb81g3yazsmrf8anrd7ffmxx";
-    })
+  buildInputs = [
+    bzip2
+    openssl
+    zlib
   ];
 
-  patchFlags = [ "-p0" ];
+  preBuild = ''
+    sed -i "s/1.6.5/${finalAttrs.version}/" dmg2img.c
+  '';
 
   installPhase = ''
-    install -D dmg2img $out/bin/dmg2img
-    install -D vfdecrypt $out/bin/vfdecrypt
+    runHook preInstall
+
+    install -Dm755 dmg2img vfdecrypt -t $out/bin
+
+    runHook postInstall
   '';
 
+  passthru.updateScript = unstableGitUpdater { };
```

Comment:
Are you sure this updater will check the develop branch?

Reply:
Yup, did `nix-shell maintainers/scripts/update.nix --argstr package dmg2img` and also tried with `passthru.updateScript = nix-update-script { extraArgs = [ "--version=branch" ]; };` instead of `unstableGitUpdater` and it was the same.
Both used the develop branch, it being the default.